phc/services/agents.py

```diff
@@ -0,0 +1,17 @@
+from typing import Dict, List, Union
+from phc.base_client import BaseClient
+
+
+class Agents(BaseClient):
+    """
+    Provides access to the PHC agents API, which allows you to call LLM-based agents.
+    """
+
+    def invoke_basic(self, body: Union[str, List[Dict]]):
+        """
+        Invokes a basic agent, which supports either a basic prompt string or a list of
+        messages which can include images as data urls.
+        """
```

Comment:
it might be a good idea to list in the docs the abac permission(s) required.

Reply:
Thanks, good idea. I'll make a follow-up PR.